src/main/java/ch/njol/skript/effects/EffReplace.java

```diff
@@ -96,43 +108,58 @@ protected void execute(Event e) {
 			return;
 		if (replaceString) {
 			if (replaceFirst) {
-				for (int x = 0; x < haystack.length; x++)
+				for (int x = 0; x < haystack.length; x++) {
```

Comment:
Is there a reason we're using `x` instead of `i`? If not, we should change all occurrences of `x` like this to `i` (probably just that it's old code I'm assuming)